pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -943,7 +943,7 @@ func TestReconcileWithTimeout(t *testing.T) {
 	// Check that the expected TaskRun was created
 	actual := clients.Pipeline.Actions()[0].(ktesting.CreateAction).GetObject().(*v1alpha1.TaskRun)
 	if actual == nil {
-		t.Errorf("Expected a TaskRun to be created, but it wasn't.")
+		t.Fatalf("Expected a TaskRun to be created, but it wasn't.")
```

Comment:
`Fatal` returns right away, otherwise, if `actual == nil`, the code will panic, on link 950.